library/CoreBundle/Resources/config/orm_target_entities.yml

```diff
@@ -97,6 +97,8 @@ doctrine:
         Ivoz\Provider\Domain\Model\FixedCost\FixedCost
       Ivoz\Provider\Domain\Model\FixedCostsRelInvoice\FixedCostsRelInvoiceInterface:
         Ivoz\Provider\Domain\Model\FixedCostsRelInvoice\FixedCostsRelInvoice
+      Ivoz\Provider\Domain\Model\FixedCostsRelInvoiceSchedulers\FixedCostsRelInvoiceSchedulersInterface:
+        Ivoz\Provider\Domain\Model\FixedCostsRelInvoiceSchedulers\FixedCostsRelInvoiceSchedulers
```

Comment:
This should be _FixedCostsRelInvoiceScheduler\FixedCostsRelInvoiceSchedulerInterface_ (without trailing _s_)